src/ui/popup.js

```diff
@@ -452,12 +454,17 @@ export default class Popup extends Evented {
      * @param {string} className Non-empty string with CSS class name to remove from popup container
      *
      * @example
-     * let popup = new mapboxgl.Popup()
-     * popup.removeClassName('some-class')
+     * const popup = new mapboxgl.Popup({className: 'some classes'})
+        .setText('Popup')
```

Comment:
Nit: I think the asterisks are necessary to continue a jsdoc comment. Same in block below

```suggestion
    *  .setText('Popup')
```

Reply:
Good catch!